template-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -19,7 +19,7 @@ object Versions {
     const val DATASTORE_PREFERENCES = "1.0.0"
     const val DETEKT = "1.21.0"
 
-    const val GRADLE = "7.4.2"
+    const val GRADLE = "8.1.0"
```

Comment:
@luongvo since Gradle 8.1.1 has been released and it is recommended to upgrade to 8.1.1 instead. Can you help to do it? 😄 [Release doc](https://docs.gradle.org/8.1.1/release-notes.html)
![image](https://github.com/nimblehq/android-templates/assets/60863885/d6540bef-0a40-4bcf-816f-dc218eda4931)

Reply:
@manh-t Updated in https://github.com/nimblehq/android-templates/pull/508/commits/d49e49f1a0e185c5a0184a0eaae75bf8524a7391